components-sdk/src/locales/es.json

```diff
@@ -0,0 +1,63 @@
+{
+  "components.content": "Texto",
+  "components.container": "Contenedor",
+  "components.image": "Imagen",
+  "components.file": "Archivo",
+  "components.separator": "Separador",
+  "components.button": "Botón",
+  "components.button-link": "Botón de enlace",
+  "components.select-menu": "Menú de selección",
+  "components.thumbnail": "Miniatura",
+  "components.text-input": "Entrada de texto",
+  "button.add-component": "Agregar componente",
+  "button.add-inline": "Añadir componente debajo",
+  "button.add-content": "Añadir contenido",
+  "button.add-button": "Añadir botón",
+  "button.new-option": "Nueva opción",
+  "error": "Error:",
+  "actions.add-action": "Añadir acción",
+  "file.upload-file": "Click para subir un archivo",
+  "separator.margin": "Haz clic para cambiar el margen",
+  "thumbnail.set-image-url": "Usar imagen de internet",
+  "thumbnail.image-url": "URL de la imagen, https://...",
+  "thumbnail.upload-image": "Subir imagen",
+  "thumbnail.add-description": "Añadir texto alternativo",
+  "thumbnail.change-description": "Cambiar texto alternativo",
+  "thumbnail.clear-description": "Borrar texto alternativo",
+  "thumbnail.set-spoiler": "Establecer spoiler",
+  "thumbnail.remove-spoiler": "Quitar spoiler",
+  "thumbnail.delete": "Eliminar",
+  "thumbnail.add-image": "Agregar nueva imagen",
+  "alttext.edit": "(Editar)",
+  "string-select.global-settings": "Configuración global",
+  "string-select.invalid": "INVÁLIDO:",
+  "string-select.mark-enabled": "Marcar todo como activado",
+  "string-select.mark-disabled": "Marcar todo como desactivado",
+  "string-select.add-placeholder": "Añadir placeholder",
+  "string-select.change-placeholder": "Cambiar placeholder",
+  "string-select.clear-placeholder": "Borrar placeholder",
+  "string-select.set-minimum": "Establecer número mínimo de opciones seleccionadas",
+  "string-select.set-maximum": "Establecer número máximo de opciones seleccionadas",
+  "string-select.default-unselect": "Deseleccionar esto por defecto",
+  "string-select.default-select": "Seleccionar esto por defecto",
```

Comment:
```suggestion
  "string-select.default-unselect": "Deseleccionar como defecto",
  "string-select.default-select": "Seleccionar como defecto",
```

Reply:
"Deseleccionar como defecto" is indeed weird, but "Quitar como defecto" is also a weird translation for me.

"Seleccionar como defecto" is a perfectly fine translation and also makes sense grammatically.

"Seleccionar por defecto" _would_ be fine but, it leads to think that its being selected as the default, not that it will be selected as the default one for people. I would be fine with this translation, but "como" makes more sense to me.

_Should also highlight, I am from Spain, so what makes sense intuition wise might differ from yours if you are not from there._